### PR TITLE
units: add a basic SystemCallFilter

### PR DESCRIPTION
**Pull request by topimiettinen**
Add a line
SystemCallFilter=~@clock @module @mount @obsolete @raw-io ptrace
for daemons shipped by systemd. As an exception, systemd-timesyncd
needs @clock system calls and systemd-localed is not privileged.
ptrace(2) is blocked to prevent seccomp escapes.

### CODE REVIEW

**Comment by poettering** [comment]
I presume you gave this some basic testing? Patch looks fine. (Though I probably would not have placed the `@privileged` one in localed's unit file). I am not sure the tests actually cover seccomp correctly, so we'll see whether this explodes or not...

**Comment by poettering** [comment]
Hmm the ubuntu CI thing appears to be unrelated. @martinpitt any chance you can restart that test?

(also, do you test with seccomp support on ubuntu?)

**Comment by topimiettinen** [comment]
I'm using this list for most daemons here with success, but I don't have all systemd components installed.

The end of the autopkgtest reports some testing with the units:
boot-smoke           SKIP Test requires machine-level isolation but testbed does not provide that
timedated            PASS
hostnamed            PASS
localed-locale       PASS
localed-x11-keymap   PASS
unit-config          PASS
networkd             PASS
build-login          PASS

**Comment by martinpitt** [comment]
> Hmm the ubuntu CI thing appears to be unrelated. @martinpitt any chance you can restart that test?

Meh, transient network failure on git clone, restarted.

(also, do you test with seccomp support on ubuntu?)

I have a really basic test in 
http://anonscm.debian.org/cgit/pkg-systemd/systemd.git/tree/debian/tests/boot-and-services#n408 for custom syscall filtering, and our packages do get built with seccomp. So the components which have tests (localed, networkd, etc.) are covered; I don't have a direct functionality test of timesyncd, but the "no failed units" test in boot-and-services would at least notice if timesyncd fails to start.

So in summary: If CI is green, then this should be good.
